conf.py

```diff
@@ -307,7 +307,7 @@ def get_tool_name(envvar, default):
     if re.search(r'[^-/_+.A-Za-z0-9]', tool):
         # Or we could actually escape it...
         print("Bad characters in $%s" % envvar)
-        sys.exit(1)
+#        sys.exit(1)
```

Comment:
It could be the hyphens when we we are setting  `-classpath` and `-Djava.library.path`. There isn't an issue now since we are setting the JAVA_BINDINGS_DIR and generating the classpath and library path from that.